README.md

```diff
@@ -1,2 +1,32 @@
 # TOL Api
 A node.js API client for the TOL APIs.
+
+###Requirements
+The api client requires [q](https://github.com/kriskowal/q), [request](https://github.com/request/request), and [underscore.js](http://underscorejs.org/).
+
+###Installation
+There are 2 ways you can import this project.
+
+1.) ```npm install tol-api```
+
+
+2.) Add the following entry to your packages.json and then run ```npm update``` (Recommended)
+
+```json
+"dependencies" :{
```

Comment:
I would suggest for those who are less familiar with node that you include an example that will work as a straight copy-and-paste:

```json
{
  "dependencies": {
    "tol-api": "0.7.x"
  }
}
```